views/posters.mustache

```diff
@@ -0,0 +1,12 @@
+{{extends layout}}
+{{block content}}
```

Comment:
please remove this. Mustache does not have block and extend. It is more painful that way. Please pull from master to see the latest code.

Reply:
Updated it.